app/controllers/spree/user_sessions_controller.rb

```diff
@@ -60,11 +55,11 @@ def email_unconfirmed?
     end
 
     def render_unconfirmed_response
-      render status: :unprocessable_entity, cable_ready: cable_car.inner_html(
-        "#login-feedback",
-        partial("layouts/alert", locals: { type: "alert", message: t(:email_unconfirmed),
-                                           unconfirmed: true, tab: "login" })
-      )
+      message = t(:email_unconfirmed)
+      @local_values = { type: "alert", message:, unconfirmed: true,
+                        tab: "login", email: params.dig(:spree_user, :email) }
+
+      render :unconfirmed_user, status: :unprocessable_entity
```

Comment:
FYI It probably would have been simpler to define the turbo stream directly here, instead of creating a new file. For example: https://turbo.hotwired.dev/handbook/streams#reusing-server-side-templates

But this is fine as it is 👍